.github/fabricbot.json

```diff
@@ -0,0 +1,1553 @@
+[
+    {
+      "taskType": "trigger",
```

Comment:
This is going to give you the full set of rules that exist across the language repositories - which you may or may not want.     You're also going to get the "Service Attention" and "PR Review" contacts from the repository that you grabbed this from, since FabricBot intermingles behavior and data.

We may want to scope this down together to just the rules that you'd like.   Either way, we'll need to sync labels in the azure-dev repository to the common set-in order for rules to work.   If we scope down to a subset, we can also scope down the labels that need to be added.

Feel free to grab a block on my calendar, if you want to talk through things.

Reply:
I used C++ repo as reference and remove some triggers/functionality from it.
I basically left the logic around the `issue-adressed` label *but* also took the auto-scripts for stale PR/issues and some other nice routine-cleaning tasks which are always appreciated